src/Libraries/RevitNodes/Elements/Views/View.cs

```diff
@@ -166,15 +165,15 @@ public override string ToString()
         /// </summary>
         /// <param name="parameterFilter">Parameter filter</param>
         /// <returns name="view">View</returns>
-        public Element AddFilter(Revit.Filter.ParameterFilterElement parameterFilter)
+        public Revit.Elements.Views.View AddFilter(Revit.Filter.ParameterFilterElement parameterFilter)
```

Comment:
I just tested that by creating and adding a ViewFilter to a View using Dynamo 1.2 out of the box. Then I built this, swapped DLLs, re-opened the same definition and it still worked just fine. Only the description and output name changes which seems to not have any effect on resolution of the graph. Are we good?